README.md

```diff
@@ -171,21 +171,36 @@ MONGODB_URI="mongodb://localhost:27017,localhost:27018,localhost:27019/?replicaS
 
 ### Testing Auth and TLS
 
-To test authentication and TLS, first set up a MongoDB cluster with auth and TLS configured. Testing authentication requires a user with the `root` role on the `admin` database. The Go Driver repository comes with example certificates in the `data/certificates` directory. These certs can be used for testing. Here is an example command that would run a mongod with TLS correctly configured for tests:
+To test authentication and TLS, first set up a MongoDB cluster with auth and TLS configured. Testing authentication requires a user with the `root` role on the `admin` database. Here is an example command that would run a mongod with TLS correctly configured for tests. Either set or replace PATH_TO_SERVER_KEY_FILE and PATH_TO_CA_FILE with paths to their respective files:
 
 ```
 mongod \
 --auth \
 --tlsMode requireTLS \
---tlsCertificateKeyFile $(pwd)/data/certificates/server.pem \
---tlsCAFile $(pwd)/data/certificates/ca.pem \
+--tlsCertificateKeyFile $(PATH_TO_SERVER_KEY_FILE) \
```

Comment:
Minor: `$()` is a subshell in bash. Though this is a placeholder, it may be clearer to use `$PATH_TO_SERVER_KEY_FILE` and similarly for the others.